pkgdown/_pkgdown.yml

```diff
@@ -40,6 +40,7 @@ reference:
 - title: Build
   desc:  Build a complete site or its individual section components.
   contents:
+  - rlang::is_installed
```

Comment:
I'll remove this after the PR is approved.

Reply:
I think you could move it to the test section and then leave it.